service-api/module/Application/src/Model/Entity/CaseData.php

```diff
@@ -119,11 +123,10 @@ public static function fromArray(mixed $data): self
      *     documentComplete: bool,
      *     alternateAddress?: string[],
      *     searchPostcode?: string,
-     *     idMethod?: string,
+     *     yotiSessionId?: string,
      *     counterService?: string[],
-     *     kbvQuestions?: string[]
```

Comment:
Why has `kbvQuestions` been removed here? It's still set (line 156)

Reply:
Discussed on Slack: because there are duplicates (another one needs to be removed on either line 122 or 128)